src/node/node_class.hpp

```diff
@@ -328,7 +328,6 @@ WrappedObject<ClassType>::WrappedObject(const Napi::CallbackInfo& info) : Napi::
 	catch (const node::Exception& e) {
 		Napi::Error error = Napi::Error::New(info.Env(), e.what());
 		copy_object(env, e.m_value, error);
-		throw error;
```

Comment:
this is a mistake I guess

Reply:
Probably a left-over from a debugging session. I will revert it.